keras_cv/models/efficientnet_lite.py

```diff
@@ -0,0 +1,649 @@
+    include_rescaling,
+    include_top,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    **kwargs,
+):
+    """Create EfficientNetLite B1 variant."""
+    return EfficientNetLite(
+        include_rescaling,
+        include_top,
+        width_coefficient=1.0,
+        depth_coefficient=1.1,
+        default_size=240,
+        dropout_rate=0.2,
+        model_name="efficientnetlite1",
+        weights=parse_weights(weights, include_top, "efficientnetlite1"),
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        **kwargs,
+    )
+
+
+def EfficientNetLiteB2(
+    include_rescaling,
+    include_top,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    **kwargs,
+):
+    """Create EfficientNetLite B2 variant."""
+    return EfficientNetLite(
+        include_rescaling,
+        include_top,
+        width_coefficient=1.1,
+        depth_coefficient=1.2,
+        default_size=260,
+        dropout_rate=0.3,
+        model_name="efficientnetlite2",
+        weights=parse_weights(weights, include_top, "efficientnetlite2"),
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        **kwargs,
+    )
+
+
+def EfficientNetLiteB3(
+    include_rescaling,
+    include_top,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    **kwargs,
+):
+    """Create EfficientNetLite B3 variant."""
+    return EfficientNetLite(
+        include_rescaling,
+        include_top,
+        width_coefficient=1.2,
+        depth_coefficient=1.4,
+        default_size=280,
+        dropout_rate=0.3,
+        model_name="efficientnetlite3",
+        weights=parse_weights(weights, include_top, "efficientnetlite3"),
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classes=classes,
+        classifier_activation=classifier_activation,
+        **kwargs,
+    )
+
+
+def EfficientNetLiteB4(
+    include_rescaling,
+    include_top,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    **kwargs,
+):
+    """Create EfficientNetLite B4 variant."""
+    return EfficientNetLite(
+        include_rescaling,
+        include_top,
+        width_coefficient=1.4,
+        depth_coefficient=1.8,
+        default_size=300,
+        dropout_rate=0.3,
+        model_name="efficientnetlite4",
+        weights=parse_weights(weights, include_top, "efficientnetlite4"),
```

Comment:
For all of these (and maybe also model_names) -- let's include the 'b' for consistency with EfficientNetV2 (e.g. `efficientnetliteb4`)

Reply:
Done. Added `b`'s.